e2e/iam/atlas_users_test.go

```diff
@@ -11,6 +11,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+//go:build e2e || iam
 // +build e2e iam
```

Comment:
[q] Is thee second line still needed here?

Reply:
no, it is not. I am going to open a follow-up PR to update the build tags to all our e2e tests, included the ones in this PR